src/public/js/Room/animation.js

```diff
@@ -70,24 +89,28 @@ export const onAnimationComplete = () => {
 
 const showAdjacentMessages = () => {
   const adjacentMessages = store.getCurrentUser().getAdjacentMessages();
-  adjacentMessages.forEach(messageRecord => $(messageRecord).trigger('adjacent'));
+  adjacentMessages.forEach((messageRecord) =>
+    $(messageRecord).trigger('adjacent')
+  );
 
   if (adjacentMessages.length === 1) {
     $('#writeMessage').trigger({
-      type: 'messageThread', 
+      type: 'messageThread',
       threadPreviousMessage: adjacentMessages[0],
     });
 
     $(adjacentMessages[0]).trigger('indicateThread');
+  } else if (adjacentMessages.length > 1) {
+    $(adjacentMessages[0]).trigger('indicateThreadForbidden');
```

Comment:
`adjacentMessages.forEach(adjacentMessageId => $(messageId).trigger('indicateThreadForbidden'));` to trigger all adjacent threads

Reply:
It's working now. Thank you!